website/util/sanitize.py

```diff
@@ -27,9 +31,12 @@ def clean_tag(data):
     return escape_html(data).replace('"', '&quot;').replace("'", '&#39')
 
 
+def is_iterable(obj):
```

Comment:
Per flowdock discussion, this can be removed in favor of `isinstance(obj, collections.Iterable)` (and `is_iterable_but_not_string` can likewise be simplified)